helix-term/Cargo.toml

```diff
@@ -10,7 +10,7 @@ repository = "https://github.com/helix-editor/helix"
 homepage = "https://helix-editor.com"
 include = ["src/**/*", "README.md"]
 default-run = "hx"
-rust-version = "1.57"
+rust-version = "1.65"
```

Comment:
Can we remove this key altogether? Looks like it's been out of date for some time anyways

Reply:
Its not needed but I think it's nice to have as it lets cargo know what MSRV we are targeting. If somebody then tries to build helix with an out of date toolchain (om Debian or if they forgot to update for a while) they will get a helpful error message telling them that their rust toolchain is out of date and needs to be updated. 

It's not super important but nice to have and not a lot of effort to maintain so I would prefer to keep it if it's ok with you